### DISCUSSION TITLE
rsync docker

### DISCUSSION THREAD

**Discussion by kimuss** [indomus/forum#3015] [GENERALE - Discussioni generiche]
salve, vorrei avviare un comando rsync su un rasp4 senza tenere un putty acceso per vedere il progresso. Avendo un ambiente docker, è possibile creare un container di rsync così da mandare i  comandi dal container? c'è un altro modo "intelligente" per avviare un rsync sul rasp da remoto, spegnere la macchina da dove si è lanciato il comando e controllare il mattino seguente l'avanzamento e/o il completamento del comando?

grazie

**Comment by snakuzzo** [comment]
cerca su google... `linux send command background`

**Comment by maxcanna** [comment]
Puoi usare `screen`

**Comment by kimuss** [comment]
grazie
